Bigtable/src/Admin/V2/Gapic/BigtableInstanceAdminGapicClient.php

```diff
@@ -567,6 +562,13 @@ public function createInstance($parent, $instanceId, $instance, $clusters, $opti
         $request->setInstance($instance);
         $request->setClusters($clusters);
 
+        $requestParams = new RequestParamsHeaderDescriptor([
+          'parent' => $request->getParent(),
+        ]);
+        $optionalArgs['userHeaders'] = isset($optionalArgs['userHeaders'])
```

Comment:
We are using the key `headers` now.

Reply:
Fixed, gapic-generator PR here: https://github.com/googleapis/gapic-generator/pull/2011